### PR TITLE
RTX

### PR DESCRIPTION
**Pull request by jmillan**

### CODE REVIEW

File: worker/src/RTC/RtpStreamRecv.cpp [hunk 1]
```diff
@@ -72,6 +72,70 @@ namespace RTC
...
+			MS_WARN_TAG(rtx, "ignoring rtx packet with invalid payload type [ssrc: %" PRIu32 " seqnr: %" PRIu16 " payload type: %" PRIu8 "]",
+					packet->GetSsrc(),
+					packet->GetSequenceNumber(),
+					packet->GetPayloadType());
+
+			return false;
+		}
+
+		// Get the rtx packet sequence number for logging purposes.
+		this->rtxSeq = packet->GetSequenceNumber();
```
Comment (jmillan) [inline worker/src/RTC/RtpStreamRecv.cpp]: Make this a local variable. It's only used here.

File: worker/src/RTC/RtpStreamRecv.cpp [hunk 2]
```diff
@@ -72,6 +72,70 @@ namespace RTC
...
+				this->rtxSsrc,
+				this->rtxSeq,
+				packet->GetSsrc(),
+				packet->GetSequenceNumber());
+
+		// Set the extended sequence number into the packet.
+		packet->SetExtendedSequenceNumber(
+		    this->cycles + static_cast<uint32_t>(packet->GetSequenceNumber()));
+
+		// Set RTP header extension ids.
```
Comment (jmillan) [inline worker/src/RTC/RtpStreamRecv.cpp]: This is duplicated; used in ReceivePacket as well. Create a private SetHeaderExtensions method.

File: worker/include/RTC/RtpStreamRecv.hpp [hunk 3]
```diff
@@ -29,7 +29,7 @@ namespace RTC
 		RTC::RTCP::ReceiverReport* GetRtcpReceiverReport();
 		void ReceiveRtcpSenderReport(RTC::RTCP::SenderReport* report);
 		void SetRtx(uint8_t payloadType, uint32_t ssrc);
-
+		void SetHeaderExtensions(RTC::RtpPacket* packet) const;
```
Comment (ibc) [inline worker/include/RTC/RtpStreamRecv.hpp]: Should `SetHeaderExtensions()` be a public method?

Comment (jmillan) [inline worker/include/RTC/RtpStreamRecv.hpp]: No, let me fix it.

**Pull request merged**